bases/mundo_bm_wdi/country/table_config.yaml

```diff
@@ -0,0 +1,510 @@
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: national_accounts_base_year
+      bigquery_type: string
+      description: National accounts base year
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: national_accounts_reference_year
+      bigquery_type: float
+      description: National accounts reference year
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: sna_price_valuation
+      bigquery_type: string
+      description: System of national accounts price evaluation
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: lending_category
+      bigquery_type: string
+      description: Define types of lending category
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: other_groups
+      bigquery_type: string
+      description: Other abbreviations
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: system_of_national_accounts
+      bigquery_type: string
+      description: Defines which year of the system of national accounts the country
+          uses
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: alternative_conversion_factor
+      bigquery_type: string
+      description: Time period of the used conversion rate
```

Comment:
a variável é do tipo período de tempo? se for da forma convencional (dia, mês, ano) é inteiro

Reply:
ela é do formato intervalo de tempo então aparecem dados como '1971–84; 2012-15'